### ISSUE TITLE
It's a good example but I have a question

### ISSUE THREAD

**Issue by CountChu** [ni79ls/har-keras-cnn#1]
Hi,

I have a question about the code in the file
20180903_Keras_HAR_WISDM_CNN_v1.0_for_medium.py

# Normalize features for training data set
df_train['x-axis'] = feature_normalize(df['x-axis'])
df_train['y-axis'] = feature_normalize(df['y-axis'])
df_train['z-axis'] = feature_normalize(df['z-axis'])

Should it be as below?
df_train['x-axis'] = feature_normalize(df_train['x-axis'])
df_train['y-axis'] = feature_normalize(df_train['y-axis'])
df_train['z-axis'] = feature_normalize(df_train['z-axis'])

Regards,
Count Chu
